gravitee-apim-console-webui/src/management/application/applications.route.ts

```diff
@@ -391,6 +391,24 @@ function applicationsConfig($stateProvider) {
           ApplicationService.getSubscribedAPI($stateParams.applicationId),
       },
     })
+    .state('management.applications.application.notifications', {
+      url: '/notifications',
+      component: 'applicationNotificationsListSettings',
+      data: {
+        docs: {
+          page: 'management-application-notifications',
```

Comment:
```suggestion
          page: 'management-application-notifications',
          useAngularMaterial: true,
```
I haven't seen it but here you should have `useAngularMaterial` . 
this prevents bootstratp or angularJs material from adding non-encapsulated css to these screens because it uses angular material 

So `md-headline ` should become useless because it's not and angular material class

Reply:
added